### PR TITLE
implemented apify for timelines

### PR DESCRIPTION
**Pull request by yhtyyar94**

### CODE REVIEW

File: apps/api/config/cron/index.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export default {
   timelines: {
     task: timeline,
     options: {
-      rule: '0 */6 * * *',
+      rule: '0 */23 * * *',
```
Comment (7alip) [inline apps/api/config/cron/index.ts:25]: Why 23 hours? Could it be `0 0 * * *` everyday at 00:00?

File: package.json [hunk 2]
```diff
@@ -41,6 +41,7 @@
     "@wsvvrijheid/config": "*",
     "@wsvvrijheid/types": "*",
     "@wsvvrijheid/ui": "*",
+    "apify-client": "^2.7.1",
```
Comment (7alip) [inline package.json:44]: If we use `apify-client` only in Strapi app, I would add it to apps/api/package.json

**Review by 7alip — COMMENTED** [review]
Excellent! Thanks

**Comment by yhtyyar94** [comment]
all possible i will make the changes

**Comment by 7alip** [comment]
Thanks, you can merge it

**Pull request merged**